src/main/resources/schemas.yml

```diff
@@ -54,4 +54,13 @@ components:
       type: object
       properties:
         message:
-          type: string
\ No newline at end of file
+          type: string
+    Post:
```

Comment:
Okey so, You have defined it here, so what for is PostModel? Delete PostModel, and replace its usages with Post